plugins/xivo-snom/8.9.3.60/entry.py

```diff
@@ -0,0 +1,36 @@
+ 
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2014-2016 Avencall
```

Comment:
Please set the correct Copyright notice. I recommend:
```
# Copyright 2017 The Wazo Authors  (see the AUTHORS file)
```

and add the copyright holder (either yourself or your company) to the `AUTHORS` file (at the root of the repo)